lib/octokit/client/actions_workflow_runs.rb

```diff
@@ -48,6 +48,31 @@ def workflow_run(repo, id, options = {})
         get "#{Repository.path repo}/actions/runs/#{id}", options
       end
 
+      # Get workflow run usage
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param run [Integer, String, Repository, Hash] A Workflow run
+      # @return [Sawyer::Resource] the number of billable minutes and total run time for a specific workflow run
+      # @see https://docs.github.com/en/rest/reference/actions#get-workflow-run-usage
+      def workflow_runs_timing(repo, run, options = {})
+        paginate "#{Repository.path repo}/actions/runs/#{run.id}/timing", options
```

Comment:
Looking at the [API docs](https://docs.github.com/en/rest/actions/workflow-runs#get-workflow-run-usage), I don't think that this endpoint is paginated, so it should just use `get` rather than `paginate`.